lib/main.dart

```diff
@@ -120,9 +117,6 @@ class ListenScreen extends StatelessWidget {
     final ButtonStyle style =
         ElevatedButton.styleFrom(textStyle: const TextStyle(fontSize: 20));
     return Scaffold(
-        appBar: AppBar(
-          title: const Text('Listen'),
-        ),
```

Comment:
Same with this one. The user seems to not be able to go back to the previous screen.

Reply:
No, I forgot to undo this. Was testing it out.